package.json

```diff
@@ -11,11 +11,11 @@
     "lib"
   ],
   "scripts": {
-    "cli": "cross-env NODE_OPTIONS=\"--loader ts-node/esm --experimental-specifier-resolution=node --experimental-fetch\" node src/cli.ts",
+    "cli": "cross-env NODE_OPTIONS=\"--import ./register-loaders.js --experimental-specifier-resolution=node --experimental-fetch\" node src/cli.ts",
     "format": "prettier --write .",
     "check-format": "prettier --check .",
     "lint": "eslint \"src/**/*\"",
-    "build": "rimraf lib && tsc",
+    "build": "rimraf lib && tsc && fix-esm-import-path lib",
```

Comment:
I had to remove the import specifier `.js` suffixes in the source files to get it to work with node-loader/babel and then ended up needing fix-esm-import-path to add them back in the `lib` folder